hoot-core-test/src/test/cpp/hoot/core/manipulators/DividedHighwayMergerTest.cpp

```diff
@@ -56,7 +57,7 @@ namespace hoot
 class DividedHighwayMergerTest : public CppUnit::TestFixture
 {
     CPPUNIT_TEST_SUITE(DividedHighwayMergerTest);
-    //CPPUNIT_TEST(allManipulationsTest);
+    //CPPUNIT_TEST(allManipulationsTest); //TODO: re-enable or remove?
```

Comment:
This one was already disabled before the changes.  But yeah, I need to create a ticket for it.